Cooper/ClientApp/src.v2/app/services/model-services/user.service.ts

```diff
@@ -0,0 +1,11 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
+
+  constructor() { }
+
+  public user: any;
```

Comment:
I hope you add user.model User and use it

Reply:
I will when I'll be doing profile page

Reply:
no problems)